6_visualize/src/prep_view_fun.R

```diff
@@ -1,7 +1,7 @@
 
 prep_view_fun <- function(view_polygon){
   plot_fun <- function(){
-    par(omi = c(0,0,0,0), mai = c(0,0,0,0))
+    par(omi = c(0,0,0,0), mai = c(0,0,0,0), bg = '#E1EDF5')
```

Comment:
parameterize this? Probably ok to leave hard coded.

Reply:
Yeah, all of the style stuff is currently hard-coded in these functions. If we move this, we'll want to move it all

Reply:
Guideline for this project:
```
if(needs to vary by storm) {
  parameterize it into viz_config.yml
} else if(it'll be the same for all storms) {
  if (it'll be used multiple places) {
    get it into one shared location
  } else {
    leave it where it is
  }
}
```